### PR TITLE
Add ARM support

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

**Comment by rawlingsj** [comment]
add to whitelist

File: glide.lock [hunk 1]
```diff
@@ -1,215 +1,16 @@
-hash: 9589cf38139f18ca918955308856f05f0f5196ca6189cd6961ffd1aff19d8ba6
```
Comment (rhuss) [inline glide.lock]: @jimmidyson I probably messed up the glide lock file (did a 'glide get  github.com/stretchr/testify/assert' for the test).

What's the canonical way to add a new dependency ?

Comment (rhuss) [inline glide.lock]: btw is the README /wrt the requirements 'golang 1.4' and 'godeps' still accurate ?

Comment (jimmidyson) [inline glide.lock]: `glide get --strip-vendor --strip-vcs github.com/stretchr/testify/assert`. Hopefully.

Comment (rhuss) [inline glide.lock]: ok, will try that. thx :)

Comment (rhuss) [inline glide.lock]: Is it that the deps are installed as git submodules ? or should they be added exploded (duplicated) ? Hmm ...

Comment (jstrachan) [inline glide.lock]: I'm just trying to upgrade openshift + k8s dependencies right now. Its a bit fiddly - I'm about to push a `Makefile` change that updates the latest glide.yaml file changes

Comment (rhuss) [inline glide.lock]: cool, so I'll wait. I just reverted my glide update stuff.

**Pull request merged**